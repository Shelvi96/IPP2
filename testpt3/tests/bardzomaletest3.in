NEW ID

7>0
57>5587
0;:4>119::516:
:2>2972
4>997582:
1266748>114376
2>7136653301
73005::>2627;:
65506429>87637;722
2;>440
@0:9;86890432339168063259
@32:84982:;35;0933;::0;4932
@9813;
@1702
@6469959;
@42;23;;461841171
@80928362180271;9
@5
@348::823:947994;575;81;30
@2:57284;6678999246;
@58837887;1;1;411554;;408192
@85:51609798
