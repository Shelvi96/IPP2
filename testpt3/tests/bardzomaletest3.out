287669619633
2573246887938
0
0
0
0
730
0
3169083468265
1468
101200000000000
0
